sympy/physics/units/definitions.py

```diff
@@ -53,7 +53,7 @@
 # gram; used to define its prefixed units
 g = gram = grams = Quantity("gram", abbrev="g")
 gram.set_dimension(mass)
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
```

Comment:
Could this be `kg/kilo`? That would require defining kg first, however, right?

Reply:
it seems right to me.